server/app/src/main/java/com/google/heartrate/wearos/app/bluetooth/BluetoothUtils.java

```diff
@@ -0,0 +1,120 @@
+package com.google.heartrate.wearos.app.bluetooth;
+
+import android.bluetooth.BluetoothAdapter;
+import android.bluetooth.BluetoothGattServer;
+import android.bluetooth.BluetoothGattServerCallback;
+import android.bluetooth.BluetoothManager;
+import android.bluetooth.le.BluetoothLeAdvertiser;
+import android.content.Context;
+import android.content.pm.PackageManager;
+
+import com.google.heartrate.wearos.app.gatt.GattException;
+
+import static android.content.Context.BLUETOOTH_SERVICE;
+
+/**
+ * Provides bluetooth related objects and checks.
+ */
+public class BluetoothUtils {
```

Comment:
Since all methods in this utils class are static I'd give this class a private constructor (see http://www.javapractices.com/topic/TopicAction.do?Id=40#:~:text=Private%20constructors%20prevent%20a%20class,classes%20containing%20only%20constants )